test/models/pipeline_run_test.rb

```diff
@@ -14,6 +14,8 @@ class PipelineRunTest < ActiveSupport::TestCase
   end
 
   test "precaching executes report_info_json" do
```

Comment:
What if we remove the test instead of skipping it? The test isn't that long, and can be reworked later on.

Reply:
done